src/renderer/dx/CustomTextRenderer.h

```diff
@@ -28,6 +29,7 @@ namespace Microsoft::Console::Render
             this->dwriteFactory = dwriteFactory;
             this->spacing = spacing;
             this->cellSize = cellSize;
+            this->targetSize = targetSize;

```

Comment:
do you, per chance, know why we went with a `this->x=x` initializer instead of a boring constructor initializer list like `DrawingContext(...): x(x), y(y), z(z)` ?

Reply:
The initial revs of the Custom Text Renderer and Custom Text Layout were blatantly taken from a sample somewhere else. So maybe it was that. Maybe it was just a previous style not removed yet. Do you want me to clean that up here? I don't mind.

Reply:
Sure! Thanks